Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,119 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
-
-export default App;
+const BUTTONS = [
+  { text: 'C', buttonStyle: 'num' },
+  { text: '7', buttonStyle: 'num' },
+  { text: '8', buttonStyle: 'num' },
+  { text: '9', buttonStyle: 'num' },
+  { text: '4', buttonStyle: 'num' },
+  { text: '5', buttonStyle: 'num' },
+  { text: '6', buttonStyle: 'num' },
+  { text: '1', buttonStyle: 'num' },
+  { text: '2', buttonStyle: 'num' },
+  { text: '3', buttonStyle: 'num' },
+  { text: '0', buttonStyle: 'num' },
+
+];
+
+const ACTIONBUTTONS = [
+  { text: '/', buttonStyle: 'action' },
+  { text: '*', buttonStyle: 'action' },
+  { text: '-', buttonStyle: 'action' },
+  { text: '+', buttonStyle: 'action' },
+  { text: '=', buttonStyle: 'action' },
+]
```

Comment:
very nice exercise with pulling the button info off of objects like this, this will give you good practice for mapping over data and creating a reusable component for each piece of that data and all the inherent practices that come with it, such as needing to create a unique key for each item